protocol-configuration/mainnetAddresses.ts

```diff
@@ -2085,4 +2085,4 @@ const MainnetAddresses: MainnetAddresses = {
   }
 };
 
-export default MainnetAddresses;
+export default MainnetContractsConfig;
```

Comment:
Why the name change? `MainnetAddresses` makes sense.

Reply:
IMO it doesn't; it's more than just addresses, it's:
- addresses
- artifactnames
- category

therefore the naming should reflect that